### ISSUE TITLE
Update terms for objects on game mode pages

### ISSUE THREAD

**Issue by Joehuu** [ppy/osu-wiki#1201]
Need the right object terms for every game mode for ppy/osu#2337. And nice if someone(me) could update it.
![](https://puu.sh/zUM6e/5f16581423.png)

**Comment by RockRoller01** [comment]
I guess you don't need them anymore since the other PR is closed? In that case you should close this issue

**Comment by Joehuu** [comment]
No, it hasn't been done yet in the wiki pages. The correct terms are in ppy/osu#2492, and it needs to be applied in the articles. Will do.

**Comment by TicClick** [comment]
to save time, the updated terms and their osu! analogs are below (taken from https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Catch/Beatmaps/CatchBeatmap.cs for example)

## osu!catch

- circle: fruit
- slider: juice stream ("drop" and "droplet" need to link to specific sections of https://osu.ppy.sh/wiki/en/Gameplay/Hit_object/Juice_stream on first occurrence)
- spinner: banana shower

## osu!mania

nothing new really

- circle: note (I saw this sometimes being referred to as a "key")
- slider: hold note

## osu!taiko

- slider: drumroll
- spinner: swell (instead of denden, which is apparently a taiko no tatsujin term)
